app/change_set_persisters/hyrax/change_set_persister.rb

```diff
@@ -48,8 +92,24 @@ def buffer_into_index
       end
     end
 
-    def with(metadata_adapter:)
-      yield self.class.new(metadata_adapter: metadata_adapter, storage_adapter: storage_adapter)
-    end
+    private
+
+      def get_ids_of_self_and_all_members(resource:, accumulator: [])
```

Comment:
This doesn't appear to be used. Please remove.

Reply:
removed